### ISSUE TITLE
Brackets 1.4 freezes on focus of js file

### ISSUE THREAD

**Issue by tomhumphrey** [adobe/brackets#11566]
Every time I open a javascript file Brackets seems to hang for about 30 seconds before being useable again. This is happening on OS X El Capitan.

**Comment by SpeedyTechie** [comment]
I had a similar issue running Brackets on Windows. I found that it was because of the large number of files in my project (primarily because I was using gulp.js and there was a large 'node_modules' folder in the directory). I solved it by using this extension to exclude my 'node_modules' folder when opening the project: https://github.com/zaggino/brackets-file-tree-exclude.

This may or may not be the same problem you're having.
